resources/README.md

```diff
@@ -0,0 +1,9 @@
+# Resources
+
+A repository for various courses, guides, blog posts that we find useful for our processes.
+
+## Courses
+
+### Rails Speed
+
+* [Rails Speed](https://www.railsspeed.com/): A full-stack course that gives you the tools to make Ruby on Rails applications faster, more scalable, and simpler to maintain. Look for it in Subvisual's Google Drive directory, or ask [@naps62](mailto:miguel@subvisual.co) if you need help finding it.
```

Comment:
Given that our guides are open to he public, does it make sense to have the text reference our internal Google Drive?

Reply:
I agree with @pfac . This should be considered internal information. At most it should be in a handbook, not in our guides.